### PR TITLE
Fix table scroll

### PR DESCRIPTION
**Pull request by mollykreis**
# Pull Request

## 🤨 Rationale

Related to #935 

## 👩‍💻 Implementation

Update the styling for the table to make the table rows scroll when there are more than can fit in the table's vertical space.

## 🧪 Testing

Manually verified that the table gets an appropriate vertical scrollbar when the table's height is restricted using `height` and `max-height`.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Review by rajsite — APPROVED** [review]

File: change/@ni-nimble-components-8e3914f8-cdb3-4a05-a43c-f5062fa29d85.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jattasNI) [inline change/@ni-nimble-components-8e3914f8-cdb3-4a05-a43c-f5062fa29d85.json:1]: According to [this comment in the linked issue](https://github.com/ni/nimble/issues/935#issuecomment-1370110263) there's a scrolling workaround that can be removed from the Blazor example app now too.

Comment (mollykreis) [reply change/@ni-nimble-components-8e3914f8-cdb3-4a05-a43c-f5062fa29d85.json:1]: Yep...working on that now 👍

**Comment by mollykreis** [comment]
@jattasNI, @rajsite -- I realized that the original approach that I implemented & you approved caused misalignment of the cell data & headers when the scrollbar was present. I've taken a new approach, so I'm reseting your reviews on this PR so that you can look at the code again if you'd like.

**Review by jattasNI — APPROVED** [review]
Tested the Angular and Blazor example apps in macOS Firefox and they both look good to me

**Review by atmgrifter00 — APPROVED** [review]

**Review by rajsite — APPROVED** [review]

**Pull request merged**